package.json

```diff
@@ -30,7 +30,7 @@
     "aws-sdk": "^2.245.1",
     "bytes": "^2.5.0",
     "d3-queue": "^3.0.1",
-    "mapnik": "^4.0.1",
+    "mapnik": "^4.4.0",
```

Comment:
If I'm not mistaken, updating to this version of mapnik will also drop support for node versions less than 10. Would this not merit a SEMVER version change for vtcomposite?

Reply:
Good point. At the same time, mapnik is only a devDep, so I'm not sure if it warrents a larger SEMVER bump.

Reply:
@springmeyer - I guess we'll need to update to n-api `node-mapnik` at some poont ^